model-engine/tests/unit/domain/test_llm_use_cases.py

```diff
@@ -410,6 +436,79 @@ async def test_get_llm_model_endpoint_use_case_raises_not_authorized(
         )
 
 
+@pytest.mark.asyncio
+async def test_update_model_endpoint_use_case_success(
+    test_api_key: str,
+    fake_model_bundle_repository,
+    fake_model_endpoint_service,
+    fake_docker_repository_image_always_exists,
+    fake_model_primitive_gateway,
+    fake_llm_artifact_gateway,
+    fake_llm_model_endpoint_service,
+    create_llm_model_endpoint_request_streaming: CreateLLMModelEndpointV1Request,
+    update_llm_model_endpoint_request: UpdateLLMModelEndpointV1Request,
```

Comment:
could we test some more cases such as changing resource requests/cases where we expect the use case to return an error?